docs/superpowers/plans/2026-04-03-nutrition-fatigue-cleanup.md

```
@@ -0,0 +1,875 @@
+- [ ] **Step 1: Run full test suite**
+
+Run: `xcodebuild test -project AmakaFlowCompanion/AmakaFlowCompanion.xcodeproj -scheme AmakaFlowCompanion -destination 'platform=iOS Simulator,name=iPhone 17' 2>&1 | tail -30`
+
+- [ ] **Step 2: Verify build for device**
+
+Run: `xcodebuild build -project AmakaFlowCompanion/AmakaFlowCompanion.xcodeproj -scheme AmakaFlowCompanion -destination generic/platform=iOS 2>&1 | tail -5`
+
+- [ ] **Step 3: Review git log**
+
+Expected commits:
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)